charts/lakefs/README.md

```diff
@@ -29,13 +30,35 @@ To install the chart with custom configuration values:
 helm install -f my-values.yaml my-lakefs lakefs/lakefs
 ```
 
-Example my-values.yaml:
+Example `my-values.yaml` using PostgreSQL:
 
 ```yaml
 secrets:
   databaseConnectionString: postgres://postgres:myPassword@my-lakefs-db.rds.amazonaws.com:5432/lakefs?search_path=lakefs
   authEncryptSecretKey: <some random string>
 lakefsConfig: |
+  database:
+    type: postgres
+  blockstore:
+    type: s3
+    s3:
+      region: us-east-1
+  gateways:
+    s3:
+      domain_name: s3.lakefs.example.com
+```
+
+Example `my-values.yaml` using DynamoDB:
+```yaml
+secrets:
+  databaseConnectionString: postgres://postgres:myPassword@my-lakefs-db.rds.amazonaws.com:5432/lakefs?search_path=lakefs
```

Comment:
Needed for dynamo?

Reply:
For dynamo as we did for the s3 bucket - we didn't add any specific secret. Dynamo users the same credentials use to access AWS bucket.

Reply:
Removed the use of connection string with dynamo db example